src/parser.rs

```diff
@@ -945,6 +939,25 @@ b=AS:500\r
 a=rtpmap:96 H264/90000\r
 a=fmtp:96 profile-level-id=TeAo;packetization-mode=1;sprop-parameter-sets=J03gKI1oBQBboQAAAwABAAADACgPFCKg,KO4BNJI=\r
 a=control:track1\r
+";
+        let _parsed = Session::parse(&sdp[..]).unwrap();
+    }
+
+    #[test]
+    fn parse_sdp_data_after_media() {
+        let sdp = b"v=0\r
+o=- 1691154453 1 IN IP4 192.168.1.100\r
+i=Pagos\r
+a=type:broadcast\r
+s=RandD2\r
+m=video 15002 RTP/AVP 97\r
+a=range:npt=0-\r
+a=rtpmap:97 H264/90000\r
+a=fmtp:97 profile-level-id=4D4029; packetization-mode=1; sprop-parameter-sets=Z01AKZZUBQHsgA==,aO44gA==\r
+a=framerate:15.000\r
+a=control:rtsp://192.168.168.20/camera1.sdp\r
+c=IN IP4 0.0.0.0\r
+t=0 0\r
```

Comment:
My question here would be whether this should apply to the media only or to the whole session. Any guesses? :)

Reply:
Great question.. I don't know! I had a look at ffmpeg for inspiration, and it doesn't seem to look at the 't' field at all. In fact, it just ignores any line it's not interested in.[1] 't' is defined to mean timing information for the session, so I think we should just assume the camera has put the fields in the wrong order, rather than using it to mean that it is timing information for just that media item.

[1] https://github.com/FFmpeg/FFmpeg/blob/d295b6b6936c9c691a4871851fdaec04d407e54b/libavformat/rtsp.c#L419

Reply:
I would've said to just ignore the `t` (or any other "wrong") line when parsing

Reply:
What do you think, would that make sense?

Reply:
Sorry I'm a bit stacked up at the moment so haven't had a chance to get to this yet. Yes I think that sounds fine. I'll try to work on it soon.